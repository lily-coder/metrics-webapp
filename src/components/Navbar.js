import React from 'react';
import { NavLink } from 'react-router-dom';

const Navbar = () => (
    <nav className='header-navbar'>
      <div className="header-text">
        <h3>Real-time Air Quality data feed</h3>
      </div>
      <ul className="header-nav-ul">
        <li className="cities">
          <NavLink to='/' exact>Cities</NavLink>
        </li>
        <li className="details">
          <NavLink to='/Details' exact>Details</NavLink>
        </li>
      </ul>
    </nav>
);

export default Navbar;