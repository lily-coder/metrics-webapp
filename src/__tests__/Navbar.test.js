import React from 'react';
import { BrowserRouter as Router } from 'react-router-dom';
import { render, screen } from '@testing-library/react';
import renderer from 'react-test-renderer';
import Navbar from '../components/Navbar.js';

describe('Nav snapshot', () => {
  test('renders correctly', () => {
    const shot = renderer
      .create(
        <Router>
          <Navbar />
        </Router>,
      )
      .toJSON();
    expect(shot).toMatchSnapshot();
  });
});
