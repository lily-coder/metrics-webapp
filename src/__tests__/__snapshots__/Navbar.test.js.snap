// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav snapshot renders correctly 1`] = `
<div
  className="nav"
>
  <div>
    <a
      href="/"
      onClick={[Function]}
    >
      &lt;
    </a>
  </div>
  <div
    className="nav-header"
  >
    <a
      href="/"
      onClick={[Function]}
    >
      COVID-19 AFRICA STATISTICS
    </a>
  </div>
</div>
`;
